nslsii/sync_experiment/sync_experiment.py

```diff
@@ -112,10 +112,16 @@ def sync_experiment(proposal_number, beamline, verbose=False, prefix=""):
         if not should_they_be_here(username, new_data_session, beamline):
             raise AuthorizationError(f"User '{username}' is not allowed to take data on proposal {new_data_session}")
 
+        pi_name = ""
+        for user in users:
+            if user.get("is_pi"):
+                pi_name = user.get("first_name", "") + " " + user.get("last_name", "")
```

Comment:
Why not f-string?
```suggestion
                pi_name = f'{user.get("first_name", "")} {user.get("last_name", "")}'
```

Reply:
ah, yes of course! I was looking at this line and something just felt so wrong, but I couldn't see why...🤦‍♂️ thanks!